rotkehlchen/tests/unit/test_globaldb.py

```diff
@@ -988,3 +989,26 @@ def test_general_cache(globaldb):
             key_parts=[GeneralCacheType.CURVE_POOL_TOKENS, '123'],
         )
         assert values_8 == values_1
+
+
+def test_packaged_db_check_for_constant_assets(globaldb):
+    """Check that if UnknownAsset is raised for asset in CONSTANT_ASSETS, check the packaged db"""
+    # delete one entry in `CONSTANT_ASSETS`
+    with globaldb.conn.write_ctx() as cursor:
+        cursor.execute('DELETE FROM assets WHERE identifier=?;', (A_LUSD.identifier,))
+        assert cursor.rowcount == 1
+    # now resolve the asset and check that no error is raised
+    lusd = A_LUSD.resolve_to_evm_token()
+    assert lusd.asset_type == AssetType.EVM_TOKEN
+    assert lusd.identifier == A_LUSD.identifier
+
+    # delete another asset and try checking its existence
+    with globaldb.conn.write_ctx() as cursor:
+        cursor.execute('DELETE FROM assets WHERE identifier=?;', (A_DAI.identifier, ))  # noqa: E501
+        assert cursor.rowcount == 1
+
+    # now check the asset type is correct and does not raise an error
+    assert A_DAI.is_evm_token()
```

Comment:
put the full condition

```suggestion
    # now check the asset type is correct and does not raise an error
    assert A_DAI.is_evm_token() is True
```